src/DDTrace/Tracer.php

```diff
@@ -384,6 +388,10 @@ public function getTracesAsArray()
         $globalTags = $this->globalConfig->getGlobalTags();
         if ($globalTags) {
             foreach ($internalSpans as &$internalSpan) {
+                // If resource is empty, we normalize it the the operation name.
+                if ($internalSpan['resource'] === null) {
+                    $internalSpan['resource'] === $internalSpan['name'];
```

Comment:
I'm assuming this is supposed to be an assignment, right?

Reply:
Well yes, and interestingly enough this hasn't been caught by tests, this means at least one more test is required or the exiting ones have to be modified. Thanks for noticing

Reply:
[Fix](https://github.com/DataDog/dd-trace-php/pull/798/commits/4f1f873a0ef5e38f5172025cba30d043cda28800) to this comment.

I was just wondering....why you need a test suite when you have @SammyK doing your code reviews? 😎